shard.yml

```diff
@@ -25,6 +25,8 @@ targets:
     main: src/precompiled_tasks/gen/page.cr
   lucky.gen.component:
     main: src/precompiled_tasks/gen/component.cr
+  lucky.gen.task:
```

Comment:
I have not been successful at actually testing this from a generated project. The sample project doesn't ever get the `bin` directory with the scripts populated. My testing for this is essentially:

```shell
random_dir $ local_lucky init.custom project_name --api --no-auth
random_dir $ cd project_name ; shards
project_name $ ls bin
ls: bin: No such file or directory
```

I think this is a bug in the `script/precompile_tasks` file, or a change in the way shard postinstall is handled. Either way, it seems out of scope for this PR.

Running it from within the lucky src directory produces this output, which leads me to believe that it will work when the problem is resolved.

```shell
lucky$ script/precompile_tasks 
Dependencies are satisfied
Building: lucky.exec
Building: lucky.watch
Building: lucky.gen.model
Building: lucky.gen.action
Building: lucky.gen.action.browser
Building: lucky.gen.action.api
Building: lucky.gen.resource.browser
Building: lucky.gen.page
Building: lucky.gen.component
Building: lucky.gen.task
Building: lucky.secret_key
```